jest.config.js

```diff
@@ -13,4 +13,5 @@ module.exports = {
     moduleNameMapper: pathsToModuleNameMapper(compilerOptions?.paths || {}, {
         prefix: "<rootDir>/",
     }),
+    testEnvironment: 'jsdom',
```

Comment:
The default environment for test is `node`. Changing it back to `jsdom` allows tests to interact with `document`, `window`, etc. Things you get in the browser.